recipes-devtools/opkg/files/0001-opkg_solver_libsolv-Pass-opkg_config-noaction-as-noa.patch

```diff
@@ -0,0 +1,30 @@
+From 852b31b13de3ab739f889345b982a82970614cf7 Mon Sep 17 00:00:00 2001
+From: Zavaczki Raul <raul.zavaczki@ni.com>
+Date: Mon, 26 Oct 2020 05:57:41 -0700
+Subject: [PATCH] opkg_solver_libsolv: Pass opkg_config->noaction as 'noaction'
+ parameter to libsolv_solver_transaction_preamble
+
+Currently, if the option for 'download first' is true, opkg will download packages even when noaction is set to false
+Changing the function call to pass opkg_config->noaction instead of 0 will fix this behaviour
+
+Signed-off-by: Zavaczki Raul <raul.zavaczki@ni.com>
```

Comment:
Could you add an `Upstream Status: Accepted...` line to the commit message trailer in the `.patch`es, similar to [this](https://github.com/ni/meta-nilrt/blob/0a38a5b5a87c4dd315c6ffedbd75f927624c6f5d/recipes-devtools/opkg/files/0002-libopkg-clear-curl-properties-on-download-error-to-p.patch#L36). It helps us to correctly handle the .patch during a rebase.